task_three_and_four/src/test/java/com/epam/webdriver/aleksandr_gostev/test/TempMailPageTest.java

```diff
@@ -0,0 +1,63 @@
+package com.epam.webdriver.aleksandr_gostev.test;
+
+import com.epam.webdriver.aleksandr_gostev.page.google_cloud.CloudGoogleHomePage;
+import com.epam.webdriver.aleksandr_gostev.page.google_cloud.EmailYourEstimatePage;
+import com.epam.webdriver.aleksandr_gostev.page.google_cloud.EstimatePage;
+import com.epam.webdriver.aleksandr_gostev.page.google_cloud.GoogleCloudPricingCalculatorPage;
+import com.epam.webdriver.aleksandr_gostev.page.temp_mail.LetterPage;
+import com.epam.webdriver.aleksandr_gostev.testDataParams.ParametersByDataProvider;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.awt.datatransfer.UnsupportedFlavorException;
+import java.io.IOException;
+import java.time.Duration;
+
+public class TempMailPageTest {
+    private WebDriver driver;
+
+    @BeforeMethod(alwaysRun = true)
+    public void browserSetup() {
+        driver = new ChromeDriver();
```

Comment:
Please create a separate class for driver that will contain methods for creating and closing driver. And also a method that would get the driver itself. I.e. you could just use these methods in pre- and postconditions which will decrease code duplications